3_navigation.ipynb

```diff
@@ -64,7 +117,26 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Let's check the information by turning the check on and off in \"Displays\" panel. The meaning of each item is as follows:\n",
+    "\n",
+    "- Map: Predefined map information.\n",
+    "\n",
+    "- global_cost_map: Cost map generated based on the predefined map.\n",
+    "\n",
+    "- local_cost_map: Cost map generated based on laser scanner information.\n",
+    "\n",
+    "Here, the cost map is a map that numerically expresses areas that are likely to pass or with obstacles, and is used for autonomous movement."
```

Comment:
"Here, the cost map is a map that numerically expresses areas where the robot is like to pass through or areas with obstacles, and is used for autonomous movement."